src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -123,6 +128,26 @@ public void linkNewLoanToOpenTransaction(LoanDTO loan) {
       });
   }
 
+  @Override
+  public void linkRenewalLoanToUpdateTransaction(LoanDTO loan) {
```

Comment:
why is it called link if the loan is already linked with the transaction. Lets rename it to something like `borrowerRenewLoan`

Reply:
borrowerRenewLoanToUpdateTransaction

Reply:
> borrowerRenewLoanToUpdateTransaction

its hard to read, lets keep it short

Reply:
Ok